src/GeneExpressionTSnePlot.js

```diff
@@ -174,33 +173,39 @@ const GeneExpressionScatterPlot = (props) => {
     />
 
   return [
-    <AtlasAutocomplete
-      key={`expression-autocomplete`}
-      wrapperClassName={`row margin-bottom-large`}
-      atlasUrl={atlasUrl}
-      suggesterEndpoint={suggesterEndpoint}
-      initialValue={geneId}
-      defaultSpecies={speciesName}
-      onSelect={ (event) => { onSelectGeneId(event) } }
-    />,
+    showControls &&
+      <AtlasAutocomplete
+        key={`expression-autocomplete`}
+        wrapperClassName={`row margin-bottom-large`}
+        atlasUrl={atlasUrl}
+        suggesterEndpoint={suggesterEndpoint}
+        initialValue={geneId}
+        defaultSpecies={speciesName}
+        onSelect={(event) => {
+          onSelectGeneId(event)
+        }}
+      />,
     //react-responsive design
-    <Desktop key={`Desktop`}>
-      {responsiveComponent(1800/2.2)}
-    </Desktop>,
-    <Tablet key={`Tablet`}>
-      {responsiveComponent(1000/2.2)}
-    </Tablet>,
-    <Mobile key={`Mobile`}>
-      {responsiveComponent(750)}
-    </Mobile>,
-    <Default key={`Default`}>
-      {responsiveComponent(350)}
-    </Default>
+    <div key={`scatter-plot`} style={ showControls ? {} : {marginTop: `13%`}}>
+      <Desktop key={`Desktop`}>
+        {responsiveComponent(1800 / 2.2)}
+      </Desktop>
+      <Tablet key={`Tablet`}>
+        {responsiveComponent(1000 / 2.2)}
+      </Tablet>
+      <Mobile key={`Mobile`}>
+        {responsiveComponent(750)}
+      </Mobile>
+      <Default key={`Default`}>
+        {responsiveComponent(350)}
+      </Default>
+    </div>
   ]
 }
 
 GeneExpressionScatterPlot.propTypes = {
   height: PropTypes.number.isRequired,
+  showControls: PropTypes.bool,
```

Comment:
The `showControls` props in `GeneExpressionTSnePlot` and `ClusterTSnePlot` should be set as `isRequired` in this case.

Reply:
Good point. Made them mandatory